jina/drivers/convert.py

```diff
@@ -139,35 +140,36 @@ def png_pack(png_tag, data):
         return 'data:image/png;base64,' + base64.b64encode(png_bytes).decode()
 
     @staticmethod
-    def image_to_byte_array(image:Image, format:str):
+    def image_to_byte_array(image: 'Image', format: str):
         import io
         img_byte_arr = io.BytesIO()
         image.save(img_byte_arr, format=format)
         img_byte_arr = img_byte_arr.getvalue()
         return img_byte_arr
 
     def png_convertor(self, arr: np.array):
+        import numpy as np
```

Comment:
Numpy is one of our key dependency and it can be put on top

Reply:
It is also kept at top. I have kept it again inside for performance as this function can be called many times. Should I remove it from the function?

Reply:
why is it better for performance to keep it inside the function?

Reply:
Han told about this earlier. 
https://github.com/jina-ai/jina/issues/373#issuecomment-623483847

Reply:
But this is about running simple `jina` commands (for instance doing `jina --help` and this kind of things) to run fast by not doing extra imports, but for the function to run fast it does not affect (or it could only be harmful). Plus, if it is alrady imported on the top.

Reply:
Okay, would like to understand why are we importing inside the functions over here?
https://github.com/jina-ai/jina-hub/blob/master/encoders/nlp/TransformerTorchEncoder/__init__.py

To confirm if importing inside helps, earlier I had tried a function with numpy import inside and outside. I think it was slightly faster when I kept it inside. I am also new to this concept of importing inside. 

As per this, it's cached the first time the function is used. So it doesn't make it slower by importing every time.
https://stackoverflow.com/questions/3095071/in-python-what-happens-when-you-import-inside-of-a-function

Reply:
It should not affect, we import inside when it is not a basic dependency (so that the fact that only importing that file do not crash jina), `numpy` is a basic dependencya and therefore it is okey

Reply:
but the main point here, is that u are importing both inside and outside. And since it is numpy let's keep it outside

Reply:
Yes I agree with the practice.